cmd/domains.go

```diff
@@ -157,68 +158,9 @@ func runDomainsCreate(cmdCtx *cmdctx.CmdContext) error {
 	return nil
 }
 
-func runDomainsRegister(cmdCtx *cmdctx.CmdContext) error {
-	ctx := cmdCtx.Command.Context()
-
-	var org *api.Organization
-	var name string
-	var err error
-
-	if len(cmdCtx.Args) == 0 {
-		org, err = selectOrganization(ctx, cmdCtx.Client.API(), "")
-		if err != nil {
-			return err
-		}
-
-		prompt := &survey.Input{Message: "Domain name to add"}
-		err := survey.AskOne(prompt, &name)
-		checkErr(err)
-		// TODO: Add some domain validation here
-	} else if len(cmdCtx.Args) == 2 {
-		org, err = cmdCtx.Client.API().GetOrganizationBySlug(ctx, cmdCtx.Args[0])
-		if err != nil {
-			return err
-		}
-		name = cmdCtx.Args[1]
-	} else {
-		return errors.New("specify all arguments (or no arguments to be prompted)")
-	}
-
-	checkResult, err := cmdCtx.Client.API().CheckDomain(ctx, name)
-	if err != nil {
-		return err
-	}
-
-	if !checkResult.RegistrationSupported {
-		return fmt.Errorf("The domain %s is not supported at this time", checkResult.DomainName)
-	}
-
-	if !checkResult.RegistrationAvailable {
-		if checkResult.TransferAvailable {
-			return fmt.Errorf("The domain %s is not available but can be transferred", checkResult.DomainName)
-		}
-		return fmt.Errorf("The domain %s is not available", checkResult.DomainName)
-	}
-
-	formattedCost := humanize.FormatFloat("", float64(checkResult.RegistrationPrice)/100)
-
-	fmt.Printf("%s is available!\n", checkResult.DomainName)
-
-	fmt.Printf("Registration costs $%s per year and will renew automatically after the first year.\n", formattedCost)
-	fmt.Println("Your account will be charged once the domain is registered. This transaction is non-refundable.")
-
-	if !confirm(fmt.Sprintf("Register %s for $%s?", name, formattedCost)) {
-		return nil
-	}
-
-	fmt.Printf("Registering domain %s in organization %s\n", name, org.Slug)
-
-	_, err = cmdCtx.Client.API().CreateAndRegisterDomain(org.ID, name)
-	if err != nil {
-		return err
-	}
+func runDomainsRegister(_ *cmdctx.CmdContext) error {
 
-	fmt.Println("Registration started")
+	fmt.Println("This command is no longer supported.")
 
 	return nil
```

Comment:
I think if we return an error here that will cause flyctl to exit with a non-zero failed exit code, which might be nice in this case (since we are in effect failing to register).